lib/msal-browser/src/crypto/BrowserCrypto.ts

```diff
@@ -77,14 +75,41 @@ export class BrowserCrypto {
     }
 
     /**
-     * Export key as given KeyFormat (see above)
+     * Export key as Json Web Key (JWK)
      * @param key 
      * @param format 
      */
     async exportJwk(key: CryptoKey): Promise<JsonWebKey> {
         return this.hasIECrypto() ? this.msCryptoExportJwk(key) : window.crypto.subtle.exportKey(KEY_FORMAT_JWK, key);
     }
 
+    /**
+     * Imports key as Json Web Key (JWK), can set extractable and usages.
+     * @param key 
+     * @param format 
+     * @param extractable 
+     * @param usages 
+     */
+    async importJwk(key: JsonWebKey, extractable: boolean, usages: Array<KeyUsage>): Promise<CryptoKey> {
+        const keyString = BrowserCrypto.getJwkString(key);
+        const keyBuffer = BrowserStringUtils.stringToArrayBuffer(keyString);
+
+        return this.hasIECrypto() ? 
+            this.msCryptoImportKey(keyBuffer, extractable, usages) 
+            : window.crypto.subtle.importKey(KEY_FORMAT_JWK, key, this._keygenAlgorithmOptions, extractable, usages);
+    }
+
+    /**
+     * Signs given data with given key
+     * @param key 
+     * @param data 
+     */
+    async sign(key: CryptoKey, data: ArrayBuffer): Promise<ArrayBuffer> {
```

Comment:
nit pick
I wonder if this could be more specific "signToken"

Reply:
I was trying to keep naming consistent with the browser crypto APIs.